src/OrbitGl/TrackManager.cpp

```diff
@@ -78,6 +78,20 @@ std::vector<FrameTrack*> TrackManager::GetFrameTracks() const {
   return tracks;
 }
 
+std::vector<Track*> TrackManager::GetTracksOnScreen() const {
+  std::vector<Track*> visible_tracks;
+  for (auto track : GetVisibleTracks()) {
+    auto track_top_y = track->GetPos()[1];
+    auto track_bottom_y = track_top_y + track->GetHeight();
+    auto screen_top_y = viewport_->GetScreenTopLeftInWorld()[1];
+    auto screen_bottom_y = screen_top_y + viewport_->GetVisibleWorldHeight();
```

Comment:
Don't use auto for simple types: https://google.github.io/styleguide/cppguide.html#Type_deduction